src/backbone.intercept.js

```diff
@@ -60,9 +60,9 @@ Backbone.Intercept = {
       return;
     }
 
-    // If the Ctrl key is held, we want to open in a new tab.
-    bypass = e.ctrlKey;
-    if (bypass) {
+    // If the platform’s modifier key is held, we want to open in a new tab.
+    var platform = navigator.platform.toLowerCase();
```

Comment:
Before we merge this in, i'd like to verify that using navigator is the current best practice here. I'd be surprised if there isn't another way (i.e.; just doing `e.metaKey || e.ctrlKey`). But if we absolutely need to do it this way, then so be it